src/views/Resources.vue

```diff
@@ -1,25 +1,64 @@
 <template>
-  <PageViewer :pageTitle="pageTitle" :content="content" />
+  <base-card class="container">
+    <h1>Resources table</h1>
+    <resource-table :resources="resources"></resource-table>
+  </base-card>
 </template>
 
 <script>
-import PageViewer from "../components/PageViewer.vue";
+import ResourceTable from "../components/Tables/ResourceTable.vue";
+import { fetchResources } from "@/services/ResourcesService.js";
 
 export default {
   components: {
-    PageViewer,
+    ResourceTable,
   },
   data() {
     return {
-      pageTitle: "Resources Page",
-      content: "This is the resources content",
+      resources: [],
     };
   },
+
+  methods: {
+    async fetchResources() {
+      try {
+        const fetchedResources = await fetchResources();
+        this.resources = fetchedResources;
+      } catch (error) {
```

Comment:
You can remove the whole try/catch having in mind we are handling the error in fetchData() method.